galleries/examples/shapes_and_collections/ellipse_arrow.py

```diff
@@ -0,0 +1,58 @@
+"""
+===================================
+Ellipse with orientation arrow demo
+===================================
+
+This demo shows how to draw an ellipse with
+an orientation arrow (clockwise or counterclockwise).
+Compare this to the :doc:`Ellipse collection example
+</gallery/shapes_and_collections/ellipse_collection>`.
+"""
+
+import matplotlib.pyplot as plt
+
+from matplotlib.markers import MarkerStyle
+from matplotlib.patches import Ellipse
+from matplotlib.transforms import Affine2D
+
+# Create a figure and axis
+fig, ax = plt.subplots(subplot_kw={"aspect": "equal"})
+
+# Define an ellipse clockwise
+center = (2, 4)
+width = 30
+height = 20
+angle = 35
+ellipse = Ellipse(
+    xy=center,
+    width=width,
+    height=height,
+    angle=angle,
```

Comment:
Let's write this more compact:

```suggestion
ellipse = Ellipse(
    xy=(2, 4),
    width=30,
    height=20,
    angle=35,
```